dashboard/src/components/ChartView/ChartReadme.tsx

```diff
@@ -13,6 +13,23 @@ interface IChartReadmeProps {
   version: string;
 }
 
+// Code from https://github.com/rexxars/react-markdown/issues/69
+function flatten(text: string, child: any): any {
+  return typeof child === "string"
+    ? text + child
+    : React.Children.toArray(child.props.children).reduce(flatten, text);
+}
+
+function HeadingRenderer(props: any) {
```

Comment:
Hmm I'm not a huge fan of having a Component inline in another component, even though this is small one. @Angelmmiguel what do you think?

Reply:
Agree with @prydonius. If the component is specific to another one, I usually keep them in the same folder:

```
CommonComponent
  - CommonComponent.test.tsx
  - CommonComponent.tsx
  - CommonComponent.css
  - SpecificComponent
    - SpecificComponent.test.tsx
    - SpecificComponent.tsx
    - SpecificComponent.css
```